creator-node/src/AsyncProcessingQueue.ts

```diff
@@ -1,36 +1,51 @@
-const { Queue, Worker } = require('bullmq')
-const { logger: genericLogger } = require('./logging')
-const config = require('./config')
-const redisClient = require('./redis')
-const { clusterUtils } = require('./utils')
+import type { Job } from 'bullmq'
+import type { Request } from 'express'
+import type { SpanContext } from '@opentelemetry/api'
+
+import { Queue, Worker } from 'bullmq'
+import { logger as genericLogger } from './logging'
+import { clusterUtils, ValuesOf } from './utils'
 
 // Processing fns
-const {
-  handleTrackContentRoute: trackContentUpload,
-  handleTranscodeAndSegment: transcodeAndSegment,
-  handleTranscodeHandOff: transcodeHandOff
-} = require('./components/tracks/tracksComponentService')
+import {
+  handleTrackContentRoute as trackContentUpload,
+  handleTranscodeAndSegment as transcodeAndSegment,
+  handleTranscodeHandOff as transcodeHandOff
+} from './components/tracks/tracksComponentService'
+import { instrumentTracing, tracing } from './tracer'
+
 const {
   processTranscodeAndSegments
 } = require('./components/tracks/trackContentUploadManager')
-const { instrumentTracing, tracing } = require('./tracer')
+const redisClient = require('./redis')
+const config = require('./config')
 
 const MAX_CONCURRENCY = 100
 const EXPIRATION_SECONDS = 86400 // 24 hours in seconds
-const PROCESS_NAMES = Object.freeze({
+
+const QUEUE_NAME = 'async-processing'
+
+const ASYNC_PROCESSING_QUEUE_HISTORY = 500
+
+export const ProcessNames = {
   trackContentUpload: 'trackContentUpload',
   transcodeAndSegment: 'transcodeAndSegment',
   processTranscodeAndSegments: 'processTranscodeAndSegments',
   transcodeHandOff: 'transcodeHandOff'
-})
-const PROCESS_STATES = Object.freeze({
+} as const
+
+export const ProcessStates = {
   IN_PROGRESS: 'IN_PROGRESS',
   DONE: 'DONE',
   FAILED: 'FAILED'
-})
-const QUEUE_NAME = 'async-processing'
+} as const
```

Comment:
these we're originally in uppercase + snake casing. is there a reason for this change?

Reply:
I was treating the variables as something closer to `enum`s rather than constants so it made sense to change it from all uppercase to snake case